config/boards/nanopim1.eos

```diff
@@ -1,7 +1,7 @@
 # Allwinner H3 quad core 1Gb SoC
 BOARD_NAME="NanoPi M1"
 BOARDFAMILY="sun8i"
-BOARD_MAINTAINER=""
+BOARD_MAINTAINER="fakeshell"
```

Comment:
I suggest to move to at least CSC status if board is confirmed working halfway decently. Having an EOS board with attached maintainer doesn't make much sense to me.

Reply:
eos is often used for - when board is known to break compilation process. I know here is not the case ... perhaps adding a rule to:

https://github.com/armbian/scripts/blob/main/.github/workflows/pull-maintainers-from-database-and-make-a-pr.yml#L83-L93

if maintainer is set for eos, rename it to csc ... and control this via BOARD_MAINTAINER ? In any case, I don't want to deal with this manually. It is too much boards.

Reply:
Or maybe call it extended support as it seems to be common when board is EOS, but someone works on it to keep it alive.